apps/hotel/README.md

```diff
@@ -0,0 +1,122 @@
+# Group Checkout / Process Manager / Reactor Example
+
+NOTE: this is not intended to be the first example you touch when exploring Equinox
+
+The purpose of this template is to demonstrate the core aspects of implementing
+a Reactor:
+- a Domain model that implements the basic behavior of the aggregates in the
+  model (without directly implementing the reaction logic itself)
+- a Reactor host project that owns the reactive element of a process, using
+  Propeller to manage the reading and dispatching of events (the Process
+  implementation itself uses the Domain layer to effect some of the reactions)
+- a [Process Manager](https://www.enterpriseintegrationpatterns.com/patterns/messaging/ProcessManager.html)
+  implemented using Equinox 
+- Unit testing reactions with a MemoryStore (without using Propeller 
+  components or wiring up subscriptions)
+- Illustrates how to have a single end-to-end scenario that involves a
+  requirement for the Reactor application to drive some aspect of the processing
+  within an application. As part of that, a single scenario implementation is
+  coupled to one of the available sets of infrastructure
+- A Concrete store integration testing scenario that uses wiring that maximizes
+  the wiring code shared with the actual Reactor Host application. In this
+  scenario, steps relying on reaction processing may require retrying parts
+  that rely on a reaction having been carried out in response to a notification
+  propagated from the store's event feed
+- while the design is intended to provide for the implementation of correct
+  behavior in the face of concurrent activity on a given `GuestStay` aggregate
+  (e.g., when two clerks concurrently try to merge a stay into different
+  `GroupCheckout`s), that's intentionally not implemented here in order to
+  allow a reader to absorb the high level structure rather than getting lost in
+  a sea of details.
+
+## Domain overview
+
+The aggregates and events are based on the scenario as outlined and implemented in
+https://github.com/oskardudycz/EventSourcing.NetCore/tree/main/Sample/HotelManagement.
+The code (in C#) and video over explains the scenario in full.
+
+Having said that, here's a brief overview of how the behavior or the system is split:
+
+- `GuestStay` represents the events tracked as part of an individual's stay in
+  the hotel. It's anticipated that the bulk of hotel stays are individually
+  managed. As such, the typical conclusion to the activities is to have a
+  normal `CheckedOut` event mark the point at which payment has been accounted
+  for and the stay no longer requires tracking
```

Comment:
maybe it needs to convey in some way that "we are simulating a typical real life scenario where the group checkout is grafted onto a running system. An alternate approach might be to split the payments and checkout aspect off in such a way that unifies group and individual checkout processes. Its the former because that's easier to follow the implementation of, and probably more realistic"